python/dgl/frame.py

```diff
@@ -183,7 +183,7 @@ def __setitem__(self, rowids, feats):
         feats : Tensor
             New features.
         """
-        self.update(idx, feats)
```

Comment:
why this kind of bug hasn't been triggered before?

Reply:
Maybe previously code in `_deprecate` is used? I think we should add some tests on this.

Reply:
I think they are used in the old `send_and_recv` functions which are no longer supported.

Reply:
Will it be triggered if someone updates graph feature manually?